actors/runtime/src/main/java/cloud/orbit/actors/Stage.java

```diff
@@ -1072,15 +1076,15 @@ public void dispose()
             @Override
             public void run()
             {
-                if (localActor.isDeactivated())
+                if (localActor.isDeactivated() || getState() != NodeCapabilities.NodeState.RUNNING)
```

Comment:
I'd recommend adding some tests to check if this works as intended.
I'm worried with actor calls being received before the node is up.
And also with race conditions while shutting down.